wes/wes_output_API.json

```diff
@@ -1,330 +1,537 @@
+        "long_description": "This file reports the MHC class I and II results.  The class I alleles are derived from the OptiType results and the class II alleles come from the xHLA results. ",
+        "filter_group": "HLA",
+        "file_purpose": "Analysis view"
+    },
+    {
+        "file_path_template": "analysis/neoantigen/{run id}/combined/{run id}.filtered.tsv",
+        "short_description": "neaontigen: list of predicted neoantigens",
+        "long_description": "The combined MHC class I and II predicted neoantigens using the pVACseq software.  The column definitions are given here (ref: https://pvactools.readthedocs.io/en/latest/pvacseq/output_files.html)",
+        "filter_group": "neoantigen",
+        "file_purpose": "Analysis view"
+    },
+    {
+        "file_path_template": "analysis/purity/{run id}/{run id}.optimalpurityvalue.txt",
+        "short_description": "tumor purity: tumor purity analysis results",
+        "long_description": "Tumor purity calculations using the FACETS software (https://github.com/mskcc/facets)..",
+        "filter_group": "purity",
+        "file_purpose": "Analysis view"
+    },
+    {
+        "file_path_template": "analysis/clonality/{run id}/{run id}_pyclone.tsv",
+        "short_description": "tumor clonality: PyClone input file generated by sequenza library (https://cran.r-project.org/web/packages/sequenza/index.html)",
+        "long_description": "Input file generated for PyClone analysis.  Sequenza was used to generate the expected file format (https://github.com/Roth-Lab/pyclone#input-format).",
+        "filter_group": "clonality",
+        "file_purpose": "Analysis view"
+    },
+    {
+        "file_path_template": "analysis/clonality/{run id}/{run id}_table.tsv",
+        "short_description": "tumor clonality: tumor clonality analysis results",
+        "long_description": "Tumor clonality results using PyClone software (https://github.com/Roth-Lab/pyclone)..",
+        "filter_group": "clonality",
+        "file_purpose": "Analysis view"
+    },
+    {
+        "file_path_template": "analysis/copynumber/{run id}/{run id}_cnvcalls.txt",
+        "short_description": "copynumber: copynumber analysis results",
+        "long_description": "Copy number variation analysis results using Sentieon CNV algorithm (https://support.sentieon.com/appnotes/cnv/)",
+        "filter_group": "copynumber",
+        "file_purpose": "Analysis view"
+    },
+    {
+        "file_path_template": "analysis/copynumber/{run id}/{run id}_cnvcalls.txt.tn.tsv",
+        "short_description": "copynumber: copynumber analysis results",
+        "long_description": "Segmented copy number variation file using Sentieon CNV algorithm (https://support.sentieon.com/appnotes/cnv/)",
+        "filter_group": "copynumber",
+        "file_purpose": "Analysis view"
+    },
+    {
+        "file_path_template": "analysis/msisensor2/{run id}/{run id}_msisensor",
```

Comment:
@lentaing can you clarify the file extension?

Reply:
for reference, txt